suzieq/poller/worker/nodes/node.py

```diff
@@ -989,21 +1029,21 @@ async def run(self):
                 try:
                     request = await self._service_queue.get()
                 except asyncio.CancelledError:
+                    self.logger.info('releasing sempahore')
                     await self._terminate()
                     return
 
                 if request:
                     tasks.append(self._exec_service(
                         request[0], request[1], request[2]))
-                    self.logger.debug(
+                    self.logger.info(
```

Comment:
Why we are moving it from debug to info? I think debug was fine, it would create a lot of noise in info